contracts/potlock/src/potlock_admin_interactions.rs

```diff
@@ -0,0 +1,111 @@
+use crate::potlock_storage::{self, PotlockId, ProjectId, Status};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub type ProjectPercentage = MultiValue2<usize, u64>;
+pub const MAX_PERCENTAGE: u64 = 10_000; // 100%
+
+#[multiversx_sc::module]
+pub trait PotlockAdminInteractions:
+    potlock_storage::PotlockStorage + multiversx_sc_modules::only_admin::OnlyAdminModule
+{
+    #[only_admin]
+    #[endpoint(changeFeeForPots)]
+    fn change_fee_for_pots(&self, token_identifier: TokenIdentifier, fee: BigUint) {
```

Comment:
After a second review, I think we should not change the token_identifier more than once. So once it is set, you cannot update it any longer. So you can use set_if_empty() here, or check that the storage is empty.